### PR TITLE
Ch 771: Add goals creation to simple campaign flow

### PR DESCRIPTION
**Pull request by eshtadc**

### CODE REVIEW

File: acquia_lift.admin.inc [hunk 1]
```diff
@@ -921,6 +918,25 @@ function acquia_lift_page_variation_details_ajax_callback($form, &$form_state) {
     // first index is 1 because 0 is the control variation.
     $preview_variation_index = (!is_numeric($form_state['storage']['variation_number']) || $form_state['storage']['variation_number'] < 1) ? 1 : $form_state['storage']['variation_number'];
     $commands[] = acquia_lift_command_page_variation_preview($form_state['values']['agent'], $preview_variation_index);
+    // Otherwise the form submitted so stop creating a page variation
+    // and show a confirmation message.
+    $commands[] = acquia_lift_command_page_variation_toggle(FALSE);
+    // Show a confirmation message.
+    $current_status = personalize_agent_get_status($agent_name);
+    if ($current_status === PERSONALIZE_STATUS_NOT_STARTED) {
+      if ($settings['personalize']['campaigns'][$agent_name]['verified'] && $current_status === PERSONALIZE_STATUS_NOT_STARTED) {
```
Comment (katbailey) [inline acquia_lift.admin.inc]: No need for a second check of `$current_status === PERSONALIZE_STATUS_NOT_STARTED`

File: acquia_lift.admin.inc [hunk 2]
```diff
@@ -1137,4 +1152,481 @@ function acquia_lift_page_variation_delete($form, &$form_state, $agent_name, $va
...
+        'plugin' => $plugin,
+        'client_side' => $type == 'client' ? 1 : 0,
+        'identifier' => '',
+        'event' => $event,
+        'pages' => $form_state['values']['pages'],
+        // The data array gets populated with the plugin-specific
+        // info for the action.
+        'data' => isset($form_state['values']['options'][$plugin]) ? $form_state['values']['options'][$plugin] : array(),
+      );
+      watchdog('acquia_lift_debug', '<pre>' . print_r($form_state['values'], TRUE) . '</pre>');
```
Comment (katbailey) [inline acquia_lift.admin.inc]: Leftover debug statement?

File: acquia_lift.module [hunk 3]
```diff
@@ -1335,7 +1352,7 @@ function acquia_lift_libraries_info() {
     'vendor url' => 'http://craigsworks.com/projects/qtip/',
     'download url' => 'http://craigsworks.com/projects/qtip/download/',
     'version callback' => '_acquia_lift_libraries_get_version',
-    'variant order' => array('minified', 'source'),
+    'variant order' => array('source', 'minified', 'source'),
```
Comment (katbailey) [inline acquia_lift.module]: Should 'source' appear here twice?

File: js/acquia_lift.dom_selector.js [hunk 4]
```diff
@@ -109,12 +130,16 @@
       this.$element.unbind('click', this._onClick);
       // QTip has some problems fully removing itself so help it.
       // NOTE that QTips don't properly re-enable so disabling is not an option.
-      this.$element.find('*').each(function() {
+      this.$element.each(function() {
         if (typeof $(this).data('qtip') !== 'undefined') {
+          console.log('removed');
```
Comment (katbailey) [inline js/acquia_lift.dom_selector.js]: Should this still be here?

File: acquia_lift.admin.inc (hunk 2 above)
Comment (eshtadc) [inline acquia_lift.admin.inc]: Yup.  Will remove.

File: acquia_lift.module (hunk 3 above)
Comment (eshtadc) [inline acquia_lift.module]: Nope - more debugging...

**Pull request closed without merging**